### PR TITLE
support for dateext, having the log date in the filename

### PR DESCRIPTION
**Pull request by perusworld**
support for dateext, having the log date in the filename

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,8 @@ forever-service version 0.x.x
                                        
     --logrotateMax [value]             Maximum logrotated files to retain, default 10 (logrotate parameter)
                                        
+    --dateExt                          Archive old versions of log files adding a daily extension like YYYYMMDD instead of simply adding a number
```
Comment (arvind-agarwal) [inline README.md]: @perusworld can you rename this to logrotateDateExt, i think it is better to keep all logrotate options with logrotate prefix to be consistent

**Comment by perusworld** [comment]
I have changed it, let me know if this looks ok.

**Comment by arvind-agarwal** [comment]
@perusworld thanks, have you tested this?

**Comment by perusworld** [comment]
yup, its already pushed to our servers

**Pull request merged**